Core.ApplicationServices/Contract/ItContractService.cs

```diff
@@ -121,6 +122,49 @@ public class ItContractService : IItContractService
             return Result<IQueryable<ItContract>, OperationError>.Success(contracts);
         }
 
+        public Maybe<OperationError> RemovePaymentResponsibleUnits(int contractId, bool isInternal, IEnumerable<int> paymentIds)
+        {
+            return Modify(contractId, contract =>
+            {
+                foreach (var paymentId in paymentIds)
+                {
+                    var error = contract.ResetEconomyStreamOrganizationUnit(paymentId, isInternal);
+                    if (error.HasValue)
+                        return error.Value;
+                }
+
+                return Result<ItContract, OperationError>.Success(contract);
+            }).Match(_ => Maybe<OperationError>.None,
+                err => err);
+        }
+
+        public Maybe<OperationError> TransferPayments(int contractId, Guid targetUnitUuid, bool isInternal, IEnumerable<int> paymentIds)
+        {
```

Comment:
not covered by unit tests

Reply:
domani is not covered either... se in the comment further down what you should test at the different levels